### PR TITLE
Update CHANGEME.md

### PR DESCRIPTION
**Pull request by Heidet**
yolo

### CODE REVIEW

**Comment by ParanoidUser** [comment]
Ahoy, @Heidet! To earn the YOLO badge, please follow [these steps](https://github.com/ParanoidUser/yolo/discussions/18). Let me know if you have any questions.

File: CHANGEME.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+127 bottles of rum on the wall, 127 bottles of rum!
+You take one down, you pass it around, 127 bottles of rum on the wall.
```
Comment (ParanoidUser) [inline CHANGEME.md:2]: There should be 126 bottles.

**Review by ParanoidUser — CHANGES_REQUESTED** [review]

File: CHANGEME.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+
 126 bottles of rum on the wall, 126 bottles of rum!
```
Comment (Heidet) [inline CHANGEME.md:4]: okay

File: CHANGEME.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
 You take one down, you pass it around, 125 bottles of rum on the wall.
```
Comment (Heidet) [inline CHANGEME.md:5]: +

**Review by Heidet — COMMENTED** [review]
126

**Comment by ParanoidUser** [comment]
Seems to be a duplicate of #929.

**Pull request closed without merging**